libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.spec.ts

```diff
@@ -1877,6 +1896,42 @@ describe('Text editor', () => {
       const error = fixture.nativeElement.querySelector('sky-form-error');
       expect(error).toBeVisible();
     });
+
+    it('sets the aria-required on the iframe to true if the required validator is given', () => {
+      testComponent.formControl.markAsTouched();
+      fixture.detectChanges();
+
+      validateIframeDocumentAttribute('aria-required', 'true');
+    });
+
+    it('sets the aria-required on the iframe to false if the required validator is not given', () => {
```

Comment:
Can we add a test for if the require validator changes? I think it would be caught by the call in the status changes subscription but would be good to check.

Reply:
I can write a more specific one. The test that this comment is on already uses that `statusChanges` subscription because the form is set up with a validator that has to be removed.

Reply:
Added